--- R-3.2.0/src/gnuwin32/Makefile	2015-03-20 02:15:06.000000000 +0300
+++ R-3.2.0-success/src/gnuwin32/Makefile	2015-06-12 18:22:38.386603082 +0300
@@ -9,7 +9,7 @@
 include MkRules.rules
 include ../../share/make/vars.mk
 
-MK = $(MAKE) --no-print-directory
+MK = $(MAKE)
 
 all:
 	@$(MK) MkRules
@@ -97,7 +97,7 @@
 extra-CPPFLAGS = -I../library/grDevices/src
 
 Rpwd.exe: front-ends/Rpwd.exe
-	$(CP) $< $@
+	$(CP) ../../../../Rpwd.exe $@
 
 front-ends/Rpwd.exe:
 	$(MK) -C front-ends Rpwd
